bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -6,7 +6,6 @@
 	<type>binding</type>
 	<name>@text/addon.shelly.name</name>
 	<description>@text/addon.shelly.description</description>
-	<connection>local</connection>
```

Comment:
@markus7017 - it seems this was deleted by mistake, can you please add it again?

Reply:
fixed